animation_url/src/routes/[slug]/+page.ts

```diff
@@ -7,9 +7,7 @@ import { default as unknown } from "../../assets/unknown.svg";
 const chain = import.meta.env.DEV ? "polygon-mumbai" : "ethereum";
 const deployment = deployments[chain];
 
-const ipfsGatewayUri = import.meta.env.DEV
-  ? "https://nftstorage.link/ipfs/"
-  : "https://tableland.mypinata.cloud/ipfs/";
+const ipfsGatewayUri = "https://nftstorage.link/ipfs/";
```

Comment:
The PL managed gateways are solid these days, probably no reason to use Pinata gateway for now. I will still pin the data on Pinata just for more network support of the CIDs.